backend/portfolio42_api/models.py

```diff
@@ -1,3 +1,29 @@
+from datetime import date
 from django.db import models
+from django.core.serializers import serialize
+from django.core.serializers.json import DjangoJSONEncoder
+from django.contrib.auth.models import AbstractUser
 
-# Create your models here.
+class User(AbstractUser):
+	intra_username = models.CharField(max_length=30, unique=True)
+	intra_id = models.IntegerField(unique=True, db_index=True)
+	first_name = models.CharField(max_length=50)
+	last_name = models.CharField(max_length=50)
+	email = models.EmailField(max_length=130)
+	intra_url = models.CharField(max_length=200);
+	image_url = models.CharField(max_length=800);
+	bio = models.TextField(max_length=800)
```

Comment:
What is `bio`? Is it a small piece of text which can give a description about a user? If so, can it be blank (add `blank=True`, https://docs.djangoproject.com/en/4.2/ref/models/fields/#blank)?

Reply:
I just copied from the example I did before, I didn't check the actual model 🥲 
I had to have a User Model to implement the Authentication, but rly I would actually take a closer look to the User Model when fixing issue #2